Microsoft.Toolkit.Services/Services/OneDrive/OneDriveStorageItem.cs

```diff
@@ -36,86 +36,93 @@ public class OneDriveStorageItem
         /// <summary>
         /// Gets the date and time that the current OneDrive item was created.
         /// </summary>
-        public DateTimeOffset? DateCreated
-        {
-            get
-            {
-                return _dateCreated;
-            }
-        }
+        public DateTimeOffset? DateCreated => _dateCreated;
 
         private DateTimeOffset? _dateModified;
 
         /// <summary>
         /// Gets the date and time that the current OneDrive item was last modified.
         /// </summary>
-        public DateTimeOffset? DateModified
-        {
-            get { return _dateModified; }
-        }
+        public DateTimeOffset? DateModified => _dateModified;
 
         private string _displayName;
 
         /// <summary>
         /// Gets the user-friendly name of the current folder.
         /// </summary>
-        public string DisplayName
-        {
-            get
-            {
-                return _displayName;
-            }
-        }
+        public string DisplayName => _displayName;
 
         private string _displayType;
 
         /// <summary>
         /// Gets The user-friendly type of the item.
         /// </summary>
-        public string DisplayType
-        {
-            get
-            {
-                return _displayType;
-            }
-        }
+        public string DisplayType => _displayType;
 
         private string _folderId;
 
         /// <summary>
         /// Gets the id of the current OneDrive Item.
         /// </summary>
-        public string FolderRelativeId
-        {
-            get
-            {
-                return _folderId;
-            }
-        }
+        public string FolderRelativeId => _folderId;
 
         private string _name;
 
         /// <summary>
         /// Gets the name of the current OneDrive Item.
         /// </summary>
-        public string Name
-        {
-            get
-            {
-                return _name;
-            }
-        }
+        public string Name => _name;
 
         private string _path;
 
         /// <summary>
         /// Gets the path of the current item if the path is available
         /// </summary>
-        public string Path
+        public string Path => _path;
+
+        private long? _fileSize;
+
+        /// <summary>
+        /// Gets the size of the current item if available
+        /// </summary>
+        public long? FileSize => _fileSize;
+
+        /// <summary>
+        /// Gets the file size with reasonable formatting
+        /// </summary>
+        public string FormattedFileSize
         {
             get
             {
-                return _path;
+                var size = _fileSize.HasValue ? _fileSize.Value : 0;
+                if (size < 1024)
```

Comment:
PR #2055 was doing something similar, could we make this an extension for long?

Reply:
They started pulling it out here: https://github.com/Microsoft/WindowsCommunityToolkit/pull/2055/files#diff-e62611f48a9fc0cbaed40c2e841f0b25R30, but we should coordinate on needs/formatting.

Reply:
I think this implementation is fine the way it is here, let's pull it out in a different PR if needed